Changelog.md

```diff
@@ -13,6 +13,7 @@ Compiler Features:
 Bugfixes:
  * ABI Encoder: When encoding an empty string coming from storage do not add a superfluous empty slot for data.
  * Yul Optimizer: Do not remove ``returndatacopy`` in cases in which it might perform out-of-bounds reads that unconditionally revert as out-of-gas. Previously, any ``returndatacopy`` that wrote to memory that was never read from was removed without accounting for the out-of-bounds condition.
+ * CommonSubexpressionEliminator process assembly items in chunks with maximum size of 2000. It helps to avoid extremely time-consuming searches during code optimization.
```

Comment:
```suggestion
 * CommonSubexpressionEliminator: Process assembly items in chunks with maximum size of 2000. It helps to avoid extremely time-consuming searches during code optimization.
```

Reply:
Also needs to be sorted up one line.

Reply:
Ok, thank you for your remarks! Changelog updated accordingly.